java/stretch-renderer/README.md

```diff
@@ -1,21 +1,39 @@
-# Stretch Renderer
-A `StretchRenderer` can be used on a `RasterLayer`. A `StretchRenderer` can be used to stretch tightly grouped values across the available value spectrum, creating more contrast between objects. This is useful for identifying objects in dark images.
+# Stretch renderer
 
-![Stretch Renderer App](stretch-renderer.png)
+Use a stretch renderer to enhance the visual contrast of raster data for analysis.
+
+![Image of stretch renderer](stretch-renderer.png)
+
+## Use case
+
+An appropriate stretch renderer can enhance the contrast of raster imagery, allowing the user to control how their data is displayed for efficient imagery analysis.
 
 ## How to use the sample
-Choose one of the stretch parameter types, min max, percent clip, or standard deviation. The app constrains stretch parameter options to valid values. 
+
+Choose one of the stretch parameter types:
```

Comment:
It might be worth mentioning that the parameter options are available from the wrench icon in the action bar.